bluemira/builders/coil_supports.py

```diff
@@ -323,6 +323,7 @@ def __init__(
         super().__init__(params, build_config)
         self.tf_xz_keep_out_zone = tf_xz_keep_out_zone
         self.pf_coil_xz = pf_coil_xz
+        self.name = f"{self.SUPPORT} {self.build_config.get('name', '0')}"
```

Comment:
This now changes the use of name compared other builders (especially as this is bluemira internal). What about something like this because self.SUPPORT is the same as the default self.name with spaces in it:
```python
self.name = f"{self.name} {self.build_config.get('support_number', 0)}"
```